airsonic-taglibs/src/main/resources/META-INF/sub.tld

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
-<taglib xmlns="http://java.sun.com/xml/ns/j2ee"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
-        version="2.0">
+<taglib xmlns="http://java.sun.com/xml/ns/javaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-jsptaglibrary_2_1.xsd"
+    version="2.1">
```

Comment:
Check taglib version.

Reply:
Wmm... That's obvious with Java EE 5.

[The Java EE 5 Tutorial - Tag Library Descriptors](https://docs.oracle.com/javaee/5/tutorial/doc/bnamu.html)

Oracle has deprecated JSP technology in JSF, and the content of the 6.0 documentation is ( .. not complete?) and is provided as an external link to JSF. It's a rabbit hole.

https://docs.oracle.com/javaee/6/tutorial/doc/bnawn.html

Since there are no problems so far, it seems that descriptors will not be a big mistake even if they are derived from Java EE 5. Leave it as is for now.